sql/2019/04_Media/04_03.sql

```diff
@@ -0,0 +1,26 @@
+/*
+standard SQL
+04_03b
+
+5.25 gb query
+
+how many SVG files are requested as discrete files (5.1M)
+and breakdown of SVG filesize in 2% buckets
+
+*/
+select 
```

Comment:
There are some unused fields in this query. Could be simplified and reformatted like this:

```sql
SELECT
  APPROX_QUANTILES(count, 1000)[OFFSET(250)] AS p25,
  APPROX_QUANTILES(count, 1000)[OFFSET(500)] AS p50,
  APPROX_QUANTILES(count, 1000)[OFFSET(750)] AS p75,
  APPROX_QUANTILES(count, 1000)[OFFSET(900)] AS p90,
  COUNT(0) AS pagecount
FROM (
  SELECT
    COUNT(0) AS count
  FROM
    `httparchive.summary_requests.2019_07_01_mobile`
  WHERE
    ext = "svg"
  GROUP BY
    pageid)
```

Reply:
fixed per your suggestion